generator/cmd/generator/main.go

```diff
@@ -18,7 +19,7 @@ func (c Creator) Create(name string) (io.WriteCloser, error) {
 func main() {
 	schema := flag.String("schema", "schema.json", "plotly schema")
 	outputDirectory := flag.String("output-directory", "gen/", "output directory, must exist before generation")
-
+	*schema = "plotly-schema_v2.29.0.json"
```

Comment:
Why is this hardcoded?

Reply:
Thank you, good catch, initially experimented with different versions and kept those files with the version name around.
Removed this one.